front/src/components/nft-card-information/NftCardInformationStyle.ts

```diff
@@ -2,14 +2,57 @@ import { createStyles } from "@mantine/core";
 
 export const useStyles = createStyles((theme) => ({
   listSize: {
-    margin: [`${theme.spacing.lg}px 0 ${theme.spacing.lg}px`],
+    borderTop: "1px solid gray",
+    margin: [`${theme.spacing.xs}px 0 0`],
+    padding: [`${theme.spacing.md}px`],
+    width: "100%",
   },
 
-  priceText: {
-    marginTop: "10px",
+  gunDes: {
+    padding: [`${theme.spacing.xs}px 0 0`],
+    borderTop: "1px solid #BDBDBD",
+    margin: [`${theme.spacing.sm}px`],
   },
 
-  textDescription: {
-    marginBottom: "10px",
+  collectionText: {
+    paddong: [`${theme.spacing.xs}px  0 ${theme.spacing.xl}px`],
+  },
+
+  descriptionPanel: {
+    padding: [`${theme.spacing.md}px`],
+    boxShadow: "rgba(99, 99, 99, 0.2) 0px 2px 8px 0px",
```

Comment:
please, find similar shadow in theme.shadows and use it, or redefine shadow in theme